src/handlebars/support.handlebars

```diff
@@ -56,127 +56,134 @@
         We produce builds based upon source code at <a href="https://openjdk.java.net" target="_blank">OpenJDK</a>,
         <a href="https://www.eclipse.org/openj9/" target="_blank">Eclipse OpenJ9</a>,
         <a href="https://aws.amazon.com/corretto/" target="_blank">Amazon Corretto</a>,
-        and <a href="https://sap.github.io/SapMachine/" target="_blank">SAP Machine</a>.
+        <a href="https://sap.github.io/SapMachine/" target="_blank">SAP Machine</a>
+        and <a href="https://github.com/alibaba/dragonwell11" target="_blank">Alibaba Dragonwell</a>.
       </p>
       <p>
-        OpenJDK and Eclipse OpenJ9 provide a new feature release every six months, and a maintenance/security update based upon each active release every three months.
+        OpenJDK and Eclipse OpenJ9 provide a new feature release every six months, and a maintenance/security update based upon each
+        active release every three months. The release dates for those from the OpenJDK project are the
```

Comment:
Consider (adds "usually"):
 "The release dates for those from the OpenJDK project are the" -> "The release dates for those from the OpenJDK project are usually the"
or point people to the definitive source of info.

Reply:
I may be misinterpreting you, but the Oracle link says `They are released on the Tuesday closest to the 17th day of January, April, July and October.` and I've replicated that here - do you not consider that sufficiently definitive?

Reply:
I meant it may be safer to **refer** to that definitive source rather than **replicate** it, such that if OpenJDK update their text you won't need to notice their change and make a corresponding update the AdoptOpenJDK website.

Reply:
Hmmm I see what you're saying now, although making the text with the dates a hyperlink makes it about as clear as it can be where the definitive source is and as an end-user it would be a pain to be forced to navigate away from the site to see it while looking at the table. If it was something that I felt was likely to change I'd agree, but I do think it would be better to have that information explicit next to the table in this case. Perhaps a "Subject to change, check the link" disclaimer against it?